### PR TITLE
openssh: update to 7.9p1.

### PR DESCRIPTION
**Pull request by chneukirchen**
Please help testing!

Works on:

- [x] x86_64
- [x] x86_64-musl
- [ ] i686
- [ ] i686-musl
- [ ] armv7hf
- [ ] armv7hf-musl
- [ ] aarch64
- [ ] aarch64-musl

### CODE REVIEW

File: srcpkgs/openssh/template [hunk 1]
```diff
@@ -19,10 +19,10 @@ makedepends="libedit-devel pam-devel zlib-devel
  $(vopt_if ssl 'libressl-devel')"
 short_desc="The OpenSSH implementation of SSH protocol"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.openssh.org"
+homepage="https://www.openssh.com"
 license="BSD"
```
Comment (CameronNemo) [inline srcpkgs/openssh/template]: nit: use SPDX

Comment (maxice8) [inline srcpkgs/openssh/template]: BSD-2-Clause, ISC

**Comment by CameronNemo** [comment]
Tested on x86_64-musl with no issues so far. Ran my ansible playbook, did a normal login, and `scp`'d a file back and forth.

File: srcpkgs/openssh/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/openssh/template]: Already took care of that

**Comment by chneukirchen** [comment]
db23a2bbd2

**Pull request closed without merging**